iam-login-service/src/main/java/it/infn/mw/iam/api/scim/provisioning/ScimUserProvisioning.java

```diff
@@ -266,11 +269,22 @@ private void executePatchOperation(IamAccount account, ScimPatchOperation<ScimUs
       accountRepository.save(account);
       for (AccountUpdater u : updatesToPublish) {
         u.publishUpdateEvent(this, eventPublisher);
+        handleSpecificUpdateType(account, u);
       }
 
     }
   }
 
+  private void handleSpecificUpdateType(IamAccount account, AccountUpdater u) {
+    if (u.getType().equals(ACCOUNT_REPLACE_ACTIVE)) {
+      if (!account.isActive()) {
```

Comment:
I know this comment is extremely annoying but in these case it's commonly not used the not and you can simply switch the things to do with a if (account.isActive()) condition :)